csrc/scheduler/registry_utils.cpp

```diff
@@ -389,12 +389,32 @@ bool requiresForwardViewReplay(Fusion* fusion, ComputeAtMap& ca_map) {
   return false;
 }
 
+namespace {
+bool isSplitOnlyReshapeTransform(Fusion* fusion) {
```

Comment:
It may be slightly better to have
```
bool isSplitOnly(ViewOp* view);
```
and change line 415 to something like
```
const auto& view_ops = ir_utils::getViewOps(fusion);
if (std::all_of(view_ops.begin(), view_ops.end(), [](ViewOp* view) { return isSplitOnly(); })) {
  return false;
}
```
This would make reductionInterferingView slightly more complicated than the current PR, but the intention of checking all reshapes would be immediately clear to readers.

Reply:
thanks for the suggestion. revised.